libdirections-hybrid/src/main/java/com/mapbox/navigation/route/hybrid/MapboxHybridRouter.kt

```diff
@@ -1,18 +1,22 @@
 package com.mapbox.navigation.route.hybrid
 
-import com.mapbox.geojson.Point
```

Comment:
it looks like `com.mapbox.geojson` might be fine for the time being?

Reply:
let's discuss in voice

Reply:
discussed, restored